lib/web_ui/lib/platform_dispatcher.dart

```diff
@@ -31,7 +31,7 @@ abstract class PlatformDispatcher {
   VoidCallback? get onPlatformConfigurationChanged;
   set onPlatformConfigurationChanged(VoidCallback? callback);
 
-  Iterable<FlutterView> get views;
+  Map<Object, FlutterView> get views;
```

Comment:
We can't expose this as a Map. Maps are mutable and we don't want the framework/apps to be able to mutate this object. It's also an (unnecessary) API break. 

We have however talked about exposing a new method, something like `getViewById` that allows you to, well, get a view by passing in an ID without exposing the entire Maps interface.

Reply:
https://github.com/flutter/engine/blob/d76ccb8f44c911b2bf35813bd6fb5cb791218e89/lib/web_ui/lib/src/engine/window.dart#L49

Reply:
```dart  
@override
  Iterable<ui.FlutterView> get views => _windows.values;
  Map<Object, ui.FlutterWindow> get windows => _windows;
  final Map<Object, ui.FlutterWindow> _windows = <Object, ui.FlutterWindow>{};
```dart

Reply:
Although the views aren't currently mutable, there's places in the code where we mutate entries of `windows`. 

Ideally, I'd like to keep all the same variables as above, but having a `windows` property doesn't make sense without `FlutterWindow` and renaming it to `views` creates a conflict.